services/121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.api.service.ts

```diff
@@ -856,4 +853,8 @@ export class IntersolveVisaApiService {
     date.setMonth(date.getMonth() - 2);
     return date;
   }
+
+  private createAddressString(contactInformation: ContactInformation) {
+    return `${contactInformation.addressStreet} ${contactInformation.addressHouseNumber} ${contactInformation.addressHouseNumberAddition ?? ''}`.trim();
```

Comment:
> So you added the ?? '' to prevent an undefined address string?
Only the houseNumberAddition part, that's the part that can be undefined. The other 2 should never be undefined because in code we've already checked if they have a value.